architecture/README.md

```diff
@@ -0,0 +1,50 @@
+# NovelWriter Architecture (working draft)
+
+Novelwriter will be built using a service-oriented architecture (SOA). It will
+consist of the following services:
+
+* authentication
+* accounts (user preferences, project metadata, etc.)
+* manuscript storage/management
+* social graph engine (characters <-> settings, etc.)
```

Comment:
Generally speaking, I agree with your general architecture. However, we'll have to discuss what you mean by social graph and what they will look like as I don't think we should make the characters/settings storage complex out of the gate. That being said, if we have a simple, easy to work with solution out of the gate that will make it easy for us to add complexity and such in the future, I'm all for that.